### PR TITLE
UUID for stream API

### PR DESCRIPTION
**Pull request by ellisist**
Updated `profile-all` and `profile-pinned` endpoints to use UUIDs instead of ids, and changed tests appropriately!

### CODE REVIEW

File: socialhome/frontend/stores/streamStore.js [hunk 1]
```diff
@@ -116,14 +116,14 @@ function newRestAPI(options) {
         })
         .get({
             action: streamStoreOperations.getProfileAll,
-            path: ({id, lastId = undefined}) => `${Urls["api-streams:profile-all"]({id})}${getLastIdParam(lastId)}`,
+            path: ({uuid, lastId = undefined}) => `${Urls["api-streams:profile-all"]({uuid})}${getLastIdParam(lastId)}`,
```
Comment (jaywink) [inline socialhome/frontend/stores/streamStore.js:119]: Was trying to find where you change these being passed in but it seems that happens already in the previous PR ;)

**Review by jaywink — APPROVED** [review]
Good stuff! Thanks again :tada:

File: socialhome/frontend/stores/streamStore.js (hunk 1 above)
Comment (ellisist) [reply socialhome/frontend/stores/streamStore.js:119]: Yeah, I accidentally got ahead of myself I guess... And that actually broke fetching content from profiles, but it's fixed with this PR.

Comment (jaywink) [reply socialhome/frontend/stores/streamStore.js:119]: Well I approved it so... :) For master branch it's "move fast break things".

**Pull request merged**